.eslintrc.yml

```diff
@@ -0,0 +1,9 @@
+env:
```

Comment:
This is config for the ESLint tool. It was auto-generated and uses the defaults.